comment/templates/comment/content.html

```diff
@@ -7,9 +7,9 @@
         <small>
             <a class="{% block user_name %}{% endblock user_name %}" href="{% get_profile_url obj %}">{{ obj.user }}</a> |
             {% if obj.is_edited %}
-            <cite title="Edited">Edited: {{ obj.edited|timesince }} ago</cite>
+            <span class="font-italic" title="Edited">Edited: {{ obj.edited|timesince }} ago</span>
```

Comment:
I believe the `cite` element was here mistakenly. As far I can see [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite) , I do not think we needed it here.